tools/ci_build/github/azure-pipelines/templates/packaging-pipeline-steps.yml

```diff
@@ -80,19 +80,20 @@ steps:
     workingDirectory: $(Build.SourcesDirectory)
   displayName: "docker run build.py --wheel_file /torch_ort_src/dist/$whlfilename"
 
-- task: AzureCLI@2
-  condition: and(succeeded(), eq(variables.UploadWheel, 'yes'), eq(variables.BuildType, 'nightly'))
```

Comment:
Here UploadWheel is a parameter, not a variable. `variables.UploadWheel` would be always null. I wonder how it worked before.

Reply:
Probably before #173 , the pipeline had the two variables.